CHANGELOG.md

```diff
@@ -38,6 +38,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#3823] (https://github.com/open-telemetry/opentelemetry-python/pull/3823))
 - Add span flags to OTLP spans and links
   ([#3881](https://github.com/open-telemetry/opentelemetry-python/pull/3881))
+- Add parameters for Distros and configurators to configure autoinstrumentation
```

Comment:
> In this approach, exporters given by params would be ADDED to those from env vars. Resource attributes would be merged with those from env vars and resource detectors. Since there's only 1 sampler, param would take precedence. Same for logging_enabled.

It would be great to have this part of the description posted somewhere, though I'm not sure where. Changelog might be too concise, so just an idea:

```suggestion
- Add parameters for Distros and configurators to configure autoinstrumentation with environment variable merging
```

Reply:
will change to "in addition to existing environment variables."